contrib/inotify-tools/inotify-tools-devel-static

```diff
@@ -0,0 +1 @@
+inotify-tools-devel-static
```

Comment:
this and the symlink above are inside the inotify-tools folder and so don't actually do anything (and are merely ignored by all the tooling, so there's no errors and also nothing happens, but they shouldn't be there). a funny failure i haven't seen before i guess :D  

in general you never have to make these, just running cbuild relink-subpkgs does everything for you